tests/unit/modal/__snapshots__/modal.handleOverlayClick.spec.js.snap

```diff
@@ -1,22 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Modal: handleOverlayClick should call "onOverlayClick" prop after click on overlay 1`] = `
-<Global
-  className="ui-modal ui-modal_active"
-  noscroll={true}
->
-  <div
-    className="ui-modal__overlay"
-    onClick={[Function]}
-  />
-  <div
-    className="ui-modal__container"
-  >
-    <section
-      className="ui-modal__content"
-    >
-      Modal Content
-    </section>
-  </div>
-</Global>
-`;
+exports[`Modal: handleOverlayClick should call "onOverlayClick" prop after click on overlay 1`] = `null`;
```

Comment:
I think it's related to the same issue as above. Because it's no longer listening to the `active` prop to determine if it's active or not, this condition is met:

https://github.com/Travix-International/travix-ui-kit/pull/115/files#diff-aeb4723b4b18ec633e3c8bab2ea276e6R156

Because it is relying on the `active` prop...
https://github.com/Travix-International/travix-ui-kit/blob/fa3e4c9b54163ea12175efac8d3d3f7a82c8b55a/tests/unit/modal/modal.handleOverlayClick.spec.js#L12

(That's why this `matchSnapshot()` it's dangerous. If the developer updates the snapshot without noticing what changed, it is problematic.)

Good catch @asci !